lib/index.js

```diff
@@ -172,19 +146,31 @@ internals.parseServer = function (server, relativeTo) {
 
 internals.parsePlugin = function (plugin, relativeTo) {
 
-    plugin = Hoek.cloneWithShallow(plugin, ['options']);
-    if (typeof plugin === 'string') {
-        plugin = { register: plugin };
+    const pluginObject = { plugin };
+    if (typeof plugin.options === 'object') {
```

Comment:
'plugin' may be a string here, so its premature to reference 'plugin.options'

Reply:
Yep, the function still needs some love based on other changes still in the air :+1: